tests/compute/test_transform.py

```diff
@@ -490,6 +490,17 @@ def test_metis_partition():
     check_metis_partition(g, 2)
     check_metis_partition_with_constraint(g)
 
+@unittest.skipIf(F._default_context_str == 'gpu', reason="METIS doesn't support GPU")
+def test_hetero_metis_partition():
+    # TODO(zhengda) Metis fails to partition a small graph.
+    g = dgl.DGLGraph(create_large_graph_index(1000), readonly=True)
+    g = dgl.as_heterograph(g)
+    check_metis_partition(g, 0)
+    check_metis_partition(g, 1)
+    check_metis_partition(g, 2)
+    check_metis_partition_with_constraint(g)
```

Comment:
it's better to adapt dgl.distributed.partition_graph and run through all tests in tests/distributed/test_partition.py as well.

Reply:
Added